metrics/metrics-core/src/main/java/org/apache/servicecomb/metrics/core/meter/os/cpu/OsCpuUsage.java

```diff
@@ -19,48 +19,52 @@
 import com.netflix.spectator.api.Id;
 
 /*
- * unit : 1 jiffies = 10ms = 0.01 s
+ * unit : 1 jiffies
  * more details :
  * http://man7.org/linux/man-pages/man5/proc.5.html
  * CMD :  /proc/stat
  * cpu  2445171 599297 353967 24490633 11242   0    10780    2993             0      0
  * cpu  user    nice   system idle     iowait  irq  softirq  stealstolen      guest  guest_nice
- * 0    1       2      3      4        5        6   7        8
+ * 0    1       2      3      4        5        6   7        8                9      10
  * total = user + nice + system + idle + iowait + irq + softirq + stealstolen
  * busy = total - idle
  */
 public class OsCpuUsage extends AbstractCpuUsage {
-  private long lastTotalTime;
 
-  private long currentTotalTime;
+  private Period total = new Period();
+
+  private Period idle = new Period();
 
   public OsCpuUsage(Id id) {
-    super(id, "/proc/stat");
+    super(id);
   }
 
-  @Override
-  protected void update(String[] stats) {
-    currentTotalTime = readCurrentTotalTime(stats);
-    periodTotalTime = currentTotalTime - lastTotalTime;
-    lastTotalTime = currentTotalTime;
-
-    super.update(stats);
+  public void update() {
+    String[] stats = CpuUtils.readAndSplitFirstLine(CpuUtils.PROC_STAT);
+    if (stats == null) {
+      return;
+    }
+    update(stats);
   }
 
-  private long readCurrentTotalTime(String[] stats) {
-    long total = 0L;
-    for (int i = 1; i <= 8; i++) {
-      total += Long.parseLong(stats[i]);
+  private void update(String[] stats) {
+
+    long currentIdle = Long.parseLong(stats[4]);
+    idle.update(currentIdle);
+
+    long totalCpu = 0L;
```

Comment:
1.delete useless empty lines.
2.why not just invoke CpuUtils.summary?

    String[] stats = CpuUtils.readAndSplitFirstLine(PROC_STAT);
    total.update(CpuUtils.summary(stats, 1, 8));
    idle.update(Double.parseDouble(stats[4]));
    super.updateUsage(total.period - idle.period, total.period, false);